shared/settings/feedback-container.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+import React, {Component} from 'react'
+
+import {HeaderHoc} from '../common-adapters'
+import Feedback from './feedback'
+import logSend from '../native/log-send'
+import {connect} from 'react-redux'
+import {compose, withState, withHandlers} from 'recompose'
+
+import type {Dispatch} from '../constants/types/flux'
+
+const FeedbackWrapped = compose(
+  withState('sendLogs', 'onChangeSendLogs', true),
+  withHandlers({
+    onSendFeedbackContained: ({sendLogs, feedback, onSendFeedback}) => () => onSendFeedback(feedback, sendLogs),
+  })
+)(Feedback)
+
+type State = {
+  sentFeedback: boolean,
+  feedback: ?string,
+}
+
+class FeedbackContainer extends Component<void, {}, State> {
+  state: State;
+
+  constructor (props: {}) {
+    super(props)
+    this.state = {
+      sentFeedback: false,
+      feedback: null,
+    }
+  }
+
+  _onChangeFeedback = (feedback) => {
+    this.setState({feedback})
+  }
+
+  render () {
+    const onSendFeedback = (feedback, sendLogs) => {
+      logSend(feedback, sendLogs).then(logSendId => {
+        console.warn('logSendId is', logSendId)
+        this.setState({
```

Comment:
I think this is a dangerous pattern. you have to check to ensure you're not dismounted else setState will complain. like click send feedback and back quickly